includes/discovery/sensors/runtime/apc.inc.php

```diff
@@ -15,3 +15,65 @@
     $high_limit         = 3000;
     discover_sensor($valid['sensor'], 'runtime', $device, $oid, $index, $type, $descr, $divisor, '1', $low_limit, $low_limit_warn, $warn_limit, $high_limit, $current);
 }
+
+// InRow IRRP100
+$oids = snmp_get($device, 'airIRRP100GroupSetpointsCoolMetric.0', '-OsqnU', 'PowerNet-MIB');
+if ($oids) {
+    echo 'APC InRow IRRP100 ';
+    // airIRRP100UnitRunHoursAirFilter
+    $index = 0;
+    $cur_oid = '.1.3.6.1.4.1.318.1.1.13.3.3.1.2.3.1.';
+    $current = snmp_get($device, 'PowerNet-MIB::airIRRP100UnitRunHoursAirFilter.'.$index, '-Oqv');
```

Comment:
Please put the `PowerNet-MIB` at the end of the snmp_get call like:

`$current = snmp_get($device, 'airIRRP100UnitRunHoursAirFilter.'.$index, '-Oqv', 'PowerNet-MIB');`

Needs to be done for all the other calls as well.